po/de.po

```diff
@@ -14824,26 +14796,6 @@ msgstr "Nicht vorgemerkt"
 msgid "Select occurrence date above."
 msgstr "Wählen Sie oben ein Datum des Auftretens."
 
-#: ../src/gnome-utils/gtkbuilder/gnc-frequency.glade.h:7
```

Comment:
The following 3 removed sections are the result of https://bugzilla.gnome.org/show_bug.cgi?id=759844
Best results are currently obtained by using a separate out of tree build directory.